src/utils/date-utils.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2018 The Subscribe with Google Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Timestamp} from "../proto/api_messages";
+
+/**
+ * @param {!Date} date
+ * @return {!Timestamp}
+ */
+export function toTimestamp(date) {
+    const millis = date.getTime();
+    return new Timestamp([millis / 1000, Math.floor ((millis % 1000) * 1000000)]);
```

Comment:
Could you add a test for this file? I think the first parameter of Timestamp needs to be an integer, and it'd be good to have that checked in an automated test